bundles/binding/pom.xml

```diff
@@ -1,187 +1,188 @@
+    <module>org.openhab.binding.wemo</module>
+    <module>org.openhab.binding.weather</module>
+    <module>org.openhab.binding.ekey</module>
+    <module>org.openhab.binding.alarmdecoder</module>
+    <module>org.openhab.binding.davis</module>
+    <module>org.openhab.binding.xpl</module>
+    <module>org.openhab.binding.dscalarm</module>
+    <module>org.openhab.binding.benqprojector</module>
+    <module>org.openhab.binding.ehealth</module>
+    <module>org.openhab.binding.anel</module>
+    <module>org.openhab.binding.enigma2</module>
+    <module>org.openhab.binding.enigma2.test</module>
+    <module>org.openhab.binding.bticino</module>
+    <module>org.openhab.binding.caldav-command</module>
+    <module>org.openhab.binding.caldav-personal</module>
+    <module>org.openhab.binding.ebus</module>
+    <module>org.openhab.binding.zibase</module>
+    <module>org.openhab.binding.wago</module>
+    <module>org.openhab.binding.networkupstools</module>
+    <module>org.openhab.binding.ecobee</module>
+    <module>org.openhab.binding.autelis</module>
+    <module>org.openhab.binding.nest</module>
+    <module>org.openhab.binding.satel</module>
+    <module>org.openhab.binding.harmonyhub</module>
+    <module>org.openhab.binding.plex</module>
+    <module>org.openhab.binding.mochadx10</module>
+    <module>org.openhab.binding.denon</module>
+    <module>org.openhab.binding.mailcontrol</module>
+    <module>org.openhab.binding.primare</module>
+    <module>org.openhab.binding.frontiersiliconradio</module>
+    <module>org.openhab.binding.panasonictv</module>
+    <module>org.openhab.binding.enphaseenergy</module>
+    <module>org.openhab.binding.lightwaverf</module>
+    <module>org.openhab.binding.lightwaverf.test</module>
+    <module>org.openhab.binding.souliss</module>
+    <module>org.openhab.binding.isy</module>
+    <!--module>org.openhab.binding.sagercaster</module-->
+    <module>org.openhab.binding.ipx800</module>
+    <module>org.openhab.binding.tacmi</module>
+    <module>org.openhab.binding.rwesmarthome</module>
+    <module>org.openhab.binding.lcn</module>
+    <module>org.openhab.binding.sapp</module>
+    <module>org.openhab.binding.ucprelayboard</module>
+    <module>org.openhab.binding.rpircswitch</module>
+    <module>org.openhab.binding.myq</module>
+    <module>org.openhab.binding.panstamp</module>
+    <module>org.openhab.binding.octoller</module>
+    <module>org.openhab.binding.akm868</module>
+    <module>org.openhab.binding.sonance</module>
+    <module>org.openhab.binding.stiebelheatpump</module>
+    <module>org.openhab.binding.stiebelheatpump.test</module>
+    <module>org.openhab.binding.sallegra</module>
+    <module>org.openhab.binding.garadget</module>
+    <module>org.openhab.binding.fritzboxtr064</module>
+    <module>org.openhab.binding.smarthomatic</module>
+    <module>org.openhab.binding.gc100ir</module>
+    <module>org.openhab.binding.powermax</module>
+  </modules>
+</project>

```

Comment:
Copy a fresh version of this file from latest master and add the binding back into the list, in order to avoid this complex diff.  Same would apply to openhab_default.cfg.